admin.html

```diff
@@ -131,25 +131,25 @@ <h3 id="get-classes-head"></h3>
               <div class="col-md-4 col-sm-4 col-md-offset-4 col-sm-offset-4">
                   <h3 style="text-align: center"> Create New Class</h3>
               </div>
-                
+
             </div>
 
             <!-- New Class -->
             <div class="col-md-12 col-sm-12">
               <div class="col-md-4 col-sm-4 col-md-offset-4 col-sm-offset-4">
-                <input type="text" class="form-control class_input" id="class_input" name="class_input" required="" placeholder="Class Name"/> 
-                <p class = "error error_class_input" hidden> Class Name Already Exists </p> <br/>
-                <p class = "error empty_class_input" hidden> Class Name Can't Be Empty </p> <br/>
-                <input type="number" class="form-control group_input" id="group_input" name="group_input" required="" placeholder="Number Of Groups"/><br/>
+                <input type="text" class="form-control class_input" id="class_input" name="class_input" placeholder="Class Name"/>
```

Comment:
By "required doesn't need to have a value", I meant that you can have just an attribute, i.e. just `required` instead of `required=""`

Reply:
Oh, and there are still spaces around text in a single-line `<p>` below this line.